### ISSUE TITLE
Cause of issue #58 found

### ISSUE THREAD

**Issue by Greyvane** [PsiLupan/MakeLobbiesGreatAgain#75]
At least for me and my computer, attempting to shut down or restart the computer while MLGA is running creates the issue I described in #58 

Recreated it twice now.

**Comment by Greyvane** [comment]
Update. I had MLGA turned off, and I attempted to shut down my computer, but something called .NETFramework was 'preventing this device from shutting down'.  So I stopped the shutdown and looked at the .NETFramework error, and it said something about a process being interrupted and asked me whether or not I wanted to ignore it the error and continue running. So I said yes to ignore it, and then rebooted. 

When the computer turned back on, MLGA wouldn't work anymore. I had to uninstall NPCap again and wipe all NPCap and MLGA files (then reboot again), then reinstall again before I could use MLGA.
